travis/run_pylint.py

```diff
@@ -218,28 +218,39 @@ def get_count_fails(linter_stats, msgs_no_count=None):
         if msg not in msgs_no_count])
 
 
-def get_subpaths(paths):
+def is_installable_module(path):
+    """return False if the path doesn't contain an installable odoo module,
+    and the full path to the module manifest otherwise"""
+    manifest_path = is_module(path)
+    if manifest_path:
+        manifest = ast.literal_eval(open(manifest_path).read())
+        if manifest.get('installable', True):
+            return manifest_path
+    return False
+
+
+def get_subpaths(paths, depth=2):
     """Get list of subdirectories
     if `__init__.py` file not exists in root path then
     get subdirectories.
     Why? More info here:
         https://www.mail-archive.com/code-quality@python.org/msg00294.html
     :param paths: List of paths
+    :param depth: How many folders can opened in deep to find a module
     :return: Return list of paths with subdirectories.
     """
     subpaths = []
     for path in paths:
-        if not os.path.isfile(os.path.join(path, '__init__.py')):
-            subpaths.extend(
-                [os.path.join(path, item)
-                 for item in os.listdir(path)
-                 if os.path.isfile(os.path.join(path, item, '__init__.py')) and
-                 (not getaddons.is_module(os.path.join(path, item)) or
-                  getaddons.is_installable_module(os.path.join(path, item)))])
-        else:
-            if not getaddons.is_module(path) or \
-                    getaddons.is_installable_module(path):
-                subpaths.append(path)
+        if depth > 0:
```

Comment:
What about `if depth <= 0: continue`?
In order to reduce the nested code and see a diff cleaner

Reply:
What is faster, "not depth > 0" or "depth <= 0"?

Reply:
pylint recommend us avoid the `not`:
 - <img width="911" alt="screen shot 2018-04-11 at 14 48 53" src="https://user-images.githubusercontent.com/6644187/38639653-82039b98-3d97-11e8-93d0-1285ba745600.png">

Reply:
FYI the `timeit` test show us that both sentences spent the same time:

- <img width="735" alt="screen shot 2018-04-11 at 14 52 36" src="https://user-images.githubusercontent.com/6644187/38639811-00a8d5b2-3d98-11e8-952d-42f5e8b49e8f.png">

Reply:
removed the not